packages/cf.js/src/simple-types.ts

```diff
@@ -1,3 +1,4 @@
 export type ABIEncoding = string;
 export type AppInstanceID = string;
 export type Address = string;
+export type H256 = string;
```

Comment:
Why do we use "H256"? Seems super ambiguous compared to "bytes32" or something

Reply:
Amen.